pegasus/sites.v3/hourofcode.com/public/cookies.haml

```diff
@@ -0,0 +1,8 @@
+---
+title: Code.org Cookie Notice
+layout: wide
+---
+
+%link{ :href => "/css/cookies.css", :rel => "stylesheet" }
```

Comment:
It would be nice if `cookies.css` and `cookies.haml` had "onetrust" in the name or something, so it is clear that these are related to a specific feature and not related to all cookies.

Reply:
Good call

Reply:
in the case of `cookies.haml`, the name of the haml file defines the URL path that will take you there (code.org/cookies, hourofcode.com/cookies), so changing it to onetrust-cookies would result in navigating to `code.org/onetrust-cookies`, which feels awkward. Is there another way we could better identify these files without messing with the URL path?

Reply:
oh that's a good point. If that's the normal pattern for these "cookie" pages, then we should keep it.